src/styles/navbar-footer/navbar-footer.css

```diff
@@ -0,0 +1,14 @@
+.navbar-main {
+  background: var(--color-primary);
+  color: var(--color-white);
+  padding: 12px;
+  position: fixed;
+  bottom: 0;
+  width: 600px;
```

Comment:
If you set a `max-width` to you `body`, the elements should be as big as that. If not, then you'd set `width: 100%`. Setting fixed widths with `px` might make some issues with responsible design. Same for the header.